src/DeviceInput/deviceInputSystem.ts

```diff
@@ -35,21 +13,21 @@ export class DeviceInputSystem implements IDisposable {
     /**
      * Callback to be triggered when a device is connected
      */
-    public onDeviceConnected: (deviceName: string) => void;
+    public onDeviceConnected: (deviceType: DeviceType, deviceSlot: number) => void = () => {};

```

Comment:
I guess this means the native implementation will need to either duplicate the enum, or just use numbers directly, or consume the actual enum via napi. This seems like it complicates things a bit more for alternative implementations of `DeviceInputSystem`. Any thoughts on this @bghgary?

Reply:
Meh, I doubt this enum is going to change much, so it's probably fine to duplicate it.

Reply:
If we're okay with the current implementation.  I'll just mark this as resolved.